zerver/views/home.py

```diff
@@ -243,7 +243,8 @@ def home_real(request: HttpRequest) -> HttpResponse:
             "embedded": narrow_stream is not None,
             "invite_as": PreregistrationUser.INVITE_AS,
             "max_file_upload_size_mib": settings.MAX_FILE_UPLOAD_SIZE,
-            "giphy_api_available": bool(page_params["giphy_api_key"]),
+            "giphy_api_available": bool(page_params["giphy_api_key"])
+            and (page_params["realm_giphy_rating"] != 0),
```

Comment:
I guess this field is sorta misnamed -- the point is whether we want to display the giphy option in the compose box.

I imagine in your upcoming PR that moves the open compose box to be a handlebars template, we'll just delete these parameters and use frontend logic instead anyway.

Reply:
Yeah, removed.